CONTRIBUTING.md

```diff
@@ -148,14 +148,30 @@ During a peer review, reviewers will add a tag to the comment to help the author
 [Kudos]|The reviewer liked this, specifically. Go you.
 [Suggestion]| An optional, alternative approach.
 
-
-
 Adding multiple tags to a comment is fine.  Do not combine tags.
 
 Good: ``[Suggestion][Kudos] This is great, but...``
 
 Bad:  ``[Suggestion|Kudos] This is great, but...``
 
+
+## Pull request labels
+Labels may be applied to your pull requests, for the purposes of organization and / or overall 
+status.
+
+| label | Explanation |
```

Comment:
[Suggestion] Alphabetize the labels? Easier for people to look up the exact meaning of a label they've seen.
Also just a bit neater.

Reply:
I had organized these in the fashion of what i saw was most likely to be used to least, but if you would prefer alphabetized that can be arranged.

Reply:
resolved in 
5aa1fd6